src/main/java/mate/academy/webapp/service/impl/BookServiceImpl.java

```diff
@@ -33,8 +33,26 @@ public BookDto findById(Long id) {
 
     @Override
     public List<BookDto> getAll() {
-        return bookRepository.getAll().stream()
+        return bookRepository.findAll().stream()
                 .map(bookMapper::toDto)
                 .toList();
     }
+
+    @Override
+    public BookDto update(Long id, CreateBookRequestDto requestDto) {
```

Comment:
Is this method atomic? Think about using a transaction.

Reply:
I guess this one is still actual @dima-semeniuk